sanitiser/_truthy.js

```diff
@@ -0,0 +1,9 @@
+function isTruthy(val) {
```

Comment:
Yeah, @stephenkhess and I were talking about the duplication in the _details and _private sanitizers (they are exactly the same with details/private exchanged!). we should definitely add `truthyWithDefault` or something, that then calls isTruthy. but it seems like isTruthy does enough on its own.

I'd also be interested in other ways to reduce duplication between all our sanitizers, since they all have a lot of similar boilerplate code